SpongeAPI

```diff
@@ -1 +1 @@
-Subproject commit 14c4512bb44ae77f76c5c148d182ac81087bae31
+Subproject commit cff02b98a0bae3d6cbc0c43a6021b21b563528ab
```

Comment:
Perhaps it's https://github.com/SpongePowered/SpongeAPI/commit/a8023468dc1eca47e904d997e763929982e67c9a?